MinecraftClient/ChatBot.cs

```diff
@@ -320,6 +320,11 @@ internal void ProcessQueuedText()
         /// <param name="name">Name of the player</param>
         public virtual void OnPlayerLeave(Guid uuid, string name) { }
         
+        /// <summary>
+        /// Called when the player deaths
+        /// </summary>
+        public virtual void OnDeath() { }
```

Comment:
I think this event is already present as per #1220. Could you remove it from the PR?